src/core/hooks/use-field/use-field.ts

```diff
@@ -46,85 +46,59 @@ export const useField = <T, Err>(
 ): FieldHandle<T, Err> => {
   const { methods, state } = useFormtsContext<object, Err>(controller);
 
-  const fieldState = useMemo(() => createFieldState(state, fieldDescriptor), [
-    state,
-    impl(fieldDescriptor).__path,
-  ]);
+  const fieldState = useMemo(
+    () => createFieldState(state, fieldDescriptor, true),
+    [state, impl(fieldDescriptor).__path]
+  );
 
   useSubscription(fieldState);
 
   return createFieldHandle(fieldDescriptor, methods, fieldState, state);
 };
 
-type FieldState<T, Err> = Atom.Readonly<{
+type FieldState<T> = Atom.Readonly<{
   value: T;
   touched: TouchedValues<T>;
-  error: Err | null;
-  isValid: boolean;
-  isValidating: boolean;
 }>;
 
 const createFieldState = <T, Err>(
   state: FormtsAtomState<object, Err>,
   field: FieldDescriptor<T, Err>,
-  includeChildrenDependencies: boolean = false
-): FieldState<T, Err> => {
+  includeTriggers: boolean = false
```

Comment:
we have 2 implementations controlled by bool flag - I'd suggest splitting this into 2 functions
sth. like `createRootFieldState` and `createDescendantFieldState`

Reply:
I decided to split atom state used in `useField` into 2 separate atoms, one for `fieldState` (the same used in children) and one for `dependencies`.